cfme/cloud/keypairs.py

```diff
@@ -50,19 +49,42 @@ def wait_for_delete(self):
 
     def create(self, cancel=False):
         """Create new keypair"""
-        navigate_to(self, 'All')
-        cfg_btn('Add a new Key Pair')
+        navigate_to(self, 'Add')
         fill(keypair_form, {'name': self.name, 'provider': self.provider.name},
              action=keypair_form.save_button)
         if not cancel:
-            flash.assert_message_match('Creating Key Pair {}'.format(self.name))
+            flash.assert_no_errors()
```

Comment:
Was this flash message removed in a recent version? I'd prefer to keep at least a partial message match here or version pick the message if it hasn't been removed.